components/nocrm_io/sources/new-prospect/new-prospect.mjs

```diff
@@ -0,0 +1,26 @@
+import common from "../common/common.mjs";
+
+export default {
+  ...common,
+  name: "New Prospect",
+  version: "0.0.1",
+  key: "nocrm_io-new-prospect",
+  description: "Emit new event on each prospect created.",
+  type: "source",
+  dedupe: "unique",
+  methods: {
+    ...common.methods,
+    getWebhookEventType() {
```

Comment:
Shouldn't there be a `deploy` method in this section?

Reply:
No, because don't have a way to get a prospect list.

Reply:
As it is, this throws an error: "deploy is not implemented". How about adding in a `deploy` method just to overwrite the one that throws an error?

```
methods: {
   ...common.methods,
   deploy() {
      return true;
   },
```

Reply:
You are right! done!